webapp/src/containers/Rhs/projectList/index.tsx

```diff
@@ -63,10 +66,11 @@ const ProjectList = () => {
 
     const {data, isSuccess, isLoading} = getApiState(plugin_constants.pluginApiServiceConfigs.getAllLinkedProjectsList.apiServiceName);
     const projectsList = data as ProjectDetails[];
+    const sortedProjectList = [...projectsList].sort(sortProjectList);
```

Comment:
Is `data` always an array? I think, `data` can be `undefined` as well. And in such case, it'll break.